src/pages/standard-report/HtmlReport.js

```diff
@@ -37,26 +38,67 @@ const Loader = () => (
     </div>
 )
 
-const HtmlReport = ({ html }) =>
-    html ? (
-        <iframe
-            id="html-report-id"
-            srcDoc={wrapHtmlInTemplate(html)}
-            title="html-report-content"
-            width="100%"
-            seamless={true}
-            sandbox="allow-same-origin allow-scripts allow-modals"
-        >
-            <style jsx>{`
-                iframe {
-                    border: none;
-                    flex-grow: 1;
-                }
-            `}</style>
-        </iframe>
-    ) : (
-        <Loader />
-    )
+class HtmlReport extends Component {
+    state = { heigth: 'auto' }
+    iframeHtml = null
+    iframeBody = null
+    heightObserver = null
+
+    onIframeLoad = event => {
+        const iframe = event.target
+
+        this.iframeHtml = iframe.contentWindow.document.documentElement
+        this.iframeBody = iframe.contentWindow.document.body
+        this.heightObserver = new window.ResizeObserver(this.adjustHeight)
+
+        this.heightObserver.observe(this.iframeHtml)
+        this.heightObserver.observe(this.iframeBody)
+        this.adjustHeight()
+    }
+
+    adjustHeight = () => {
+        const height = Math.max(
+            this.iframeBody.clientHeight,
+            this.iframeBody.scrollHeight,
+            this.iframeBody.offsetHeight,
+            this.iframeHtml.clientHeight,
+            this.iframeHtml.scrollHeight,
+            this.iframeHtml.offsetHeight
```

Comment:
Usually you see the callback passed to resizeobserver get the necessary data from the entries passed to the callback (https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserverEntry). I don't think that there's a difference in doing it in this manner. But maybe good to double check.